source/js/components/i18n/catalogs-loader.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import enMessages from '../../../../network-api/locale/en/messages.js';
```

Comment:
Does lingui have the ability to write an `index.js` that does this for us, so we can say something like `import { en, de, es, fr, pl, pt } from "../../../../network-api/locale/index.js"`?

Reply:
I don’t think we can tell `lingui extract` to create a single bundle. However, we should be able to do something like this and dynamically load only the locale we need https://github.com/stereobooster/react-lingui-example/blob/master/src/i18n.js